docs/client/web-sdk/examples.mdx

```diff
@@ -0,0 +1,437 @@
+- [tabs.group](https://developer.chrome.com/docs/extensions/reference/tabs/#method-group)
+- [tabs.highlight](https://developer.chrome.com/docs/extensions/reference/tabs/#method-highlight)
+- [tabs.move](https://developer.chrome.com/docs/extensions/reference/tabs/#method-move)
+- [tabs.query](https://developer.chrome.com/docs/extensions/reference/tabs/#method-query)
+- [tabs.reload](https://developer.chrome.com/docs/extensions/reference/tabs/#method-reload)
+- [tabs.remove](https://developer.chrome.com/docs/extensions/reference/tabs/#method-remove)
+- [tabs.setZoom](https://developer.chrome.com/docs/extensions/reference/tabs/#method-setZoom)
+- [tabs.setZoomSettings](https://developer.chrome.com/docs/extensions/reference/tabs/#method-setZoomSettings)
+- [tabs.ungroup](https://developer.chrome.com/docs/extensions/reference/tabs/#method-ungroup)
+- [tabs.update](https://developer.chrome.com/docs/extensions/reference/tabs/#method-update)
+
+```html HTML
+<div style="font-family: Arial">
+	<button id="reload">Reload</button>
+	<button id="back">Go Back</button>
+	<button id="forward">Go Forward</button>
+	<button id="youtube">Open Youtube.com</button>
+	Volume: <input type="range" min="0" max="100" value="100" id="range">
+	<p>User ID: <span id="userId"></span></p>
+	<p>Current website: <span id="currentSite"></p>
+</div>
+<div id="container" style="height:720px;width:1280px"></div>
+<script type="module">
+import Hyperbeam from "https://unpkg.com/@hyperbeam/web@latest/dist/index.js"
+// TODO: Set the embedURL variable
+const embedURL = "<your-embed-url>"
+const hb = await Hyperbeam(container, embedURL)
+userId.innerText = hb.userId
+hb.tabs.onUpdated.addListener((tabId, changeInfo) => {
+	if (changeInfo.title) {
+		currentSite.innerText = changeInfo.title
+	}
+})
+// Create a tab
+const tab = await hb.tabs.create({ active: true })
+// Duplicate a tab
+const tab2 = await hb.tabs.duplicate(tab.id)
+// Set the URL of the active tab
+const updatedTab = await hb.tabs.update({url: "https://hyperbeam.dev/"})
+
+// Button click handlers
+reload.addEventListener("click", () => {
+	hb.tabs.reload()
+})
+back.addEventListener("click", () => {
+	hb.tabs.goBack()
+})
+forward.addEventListener("click", () => {
+	hb.tabs.goForward()
+})
+youtube.addEventListener("click", () => {
+	hb.tabs.update({ url: "https://youtube.com" })
+})
+range.addEventListener("change", (e) => {
+	hb.volume = e.target.value / 100
+})
+</script>
+```
+---
+## 👂 Listen to tab events
+Hyperbeam's tab event listener API mirrors Chrome extension tab API, found [here](https://developer.chrome.com/docs/extensions/reference/tabs/#event).
+
+Rather than calling `chrome.tabs.onCreated.addListener((tab) => {})`, you will would call `hb.tabs.onCreated.addListener((tab) => {})`.
+
+We support the following events:
+
+- [tabs.onActivated](https://developer.chrome.com/docs/extensions/reference/tabs/#event-onActivated)
```

Comment:
Shouldn't this part be in the reference?

Reply:
It's understandable for the code snippet to be in the example section. We should probably make the example a bit more focused (e.g. "detect the current tab has changed, query the current URL and log it")

Reply:
I checked out what you did in the reference section. On second thought this is good enough, I can't think of a better way of documenting it.